src/components/network-area-diagram-viewer/nad-viewer-parameters.ts

```diff
@@ -177,4 +185,15 @@ export class NadViewerParameters {
             NadViewerParameters.HOVER_POSITION_PRECISION_DEFAULT
         );
     }
+
+    public getEnableAdaptiveZoom(): boolean {
```

Comment:
isAdaptiveZoomEnabled would be cleaner (but you're in line with other parameters...)